<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForLocal</class>
 <widget class="QWidget" name="ForLocal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>985</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>JustTALK - Ìò∏Ïä§Ìä∏ Î£∏</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="headerFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="headerLayout">
      <property name="leftMargin">
       <number>32</number>
      </property>
      <property name="rightMargin">
       <number>32</number>
      </property>
      <item>
       <widget class="QLabel" name="logoLabel">
        <property name="styleSheet">
         <string>font-size: 18px; font-weight: bold; color: white;</string>
        </property>
        <property name="text">
         <string>üî∂ JustTALK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="roomCodeLabel">
        <property name="styleSheet">
         <string>color: #9CA3AF; font-size: 14px;</string>
        </property>
        <property name="text">
         <string>Host Room</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="headerSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="styleSheet">
         <string>color: white; font-size: 14px;</string>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="contentWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="contentLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QWidget" name="videoWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFrame" name="videoFrame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="videoLayout">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <widget class="QLabel" name="mainVideoLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>800</width>
                <height>450</height>
               </size>
              </property>
              <property name="styleSheet">
               <string>QLabel {
    background-color: #364153;
    border: 2px solid #4A5972;
    border-radius: 8px;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="videoStatusLabel">
              <property name="styleSheet">
               <string>color: #9CA3AF; font-size: 16px;</string>
              </property>
              <property name="text">
               <string>ÏõêÍ≤© Ï∞∏Í∞ÄÏûêÎ•º Í∏∞Îã§Î¶¨Îäî Ï§ë...</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="participantNameLabel">
              <property name="visible">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string>color: white;
background-color: rgba(3, 2, 19, 0.8);
padding: 8px 16px;
border-radius: 4px;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="controlFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="controlLayout">
      <property name="leftMargin">
       <number>32</number>
      </property>
      <property name="rightMargin">
       <number>32</number>
      </property>
      <item>
       <widget class="QLabel" name="timeDisplayLabel">
        <property name="styleSheet">
         <string>color: white; font-size: 14px;</string>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="leftControlSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="participantCountLabel">
        <property name="styleSheet">
         <string>color: white; font-size: 14px;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cameraButton">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Ïπ¥Î©îÎùº ÎÅÑÍ∏∞</string>
        </property>
        <property name="text">
         <string>üìπ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="muteButton">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip">
         <string>ÎßàÏù¥ÌÅ¨ ÎÅÑÍ∏∞</string>
        </property>
        <property name="text">
         <string>üé§</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="endCallButton">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip">
         <string>ÌÜµÌôî Ï¢ÖÎ£å</string>
        </property>
        <property name="text">
         <string>üìû</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="rightControlSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
